### PR TITLE
Specify minimum windows version in readme

### PR DESCRIPTION
**Pull request by Joehuu**

### CODE REVIEW

**Review by bdach — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -31,12 +31,12 @@ If you are looking to install or test osu! without setting up a development envi
...
 | ------------- | ------------- | ------------- | ------------- | ------------- |
 
 - The iOS testflight link may fill up (Apple has a hard limit of 10,000 users). We reset it occasionally when this happens. Please do not ask about this. Check back regularly for link resets or follow [peppy](https://twitter.com/ppy) on twitter for announcements of link resets.
 
-- When running on Windows 7 or 8.1, *[additional prerequisites](https://docs.microsoft.com/en-us/dotnet/core/install/windows?tabs=net50&pivots=os-windows#dependencies)** may be required to correctly run .NET 5 applications if your operating system is not up-to-date with the latest service packs.
+- When running on Windows 8.1, *[additional prerequisites](https://docs.microsoft.com/en-us/dotnet/core/install/windows?tabs=net50&pivots=os-windows#dependencies)** may be required to correctly run .NET 5 applications if your operating system is not up-to-date with the latest service packs.
```
Comment (peppy) [inline README.md:39]: not sure this is still required, best check.

Comment (Joehuu) [reply README.md:39]: I can't check as I don't have that OS.

Comment (peppy) [reply README.md:39]: Following the link, we can potentially change this to be a direct link / reference to the visual c++ redist. Not sure there would be any users on such an old OS without those installed though (windows update or steam games will likely have us covered).

Comment (Joehuu) [reply README.md:39]: I was curious so I installed windows 8.1 from https://www.microsoft.com/en-us/software-download/windows8ISO and it launched without any additional installation. Will remove.

Comment (peppy) [reply README.md:39]: Thanks for testing!

**Review by peppy — APPROVED** [review]

**Pull request merged**